packages/odyssey-react/src/components/Button/Button.module.scss

```diff
@@ -124,31 +124,18 @@
   color: var(--DismissTextColor);
   line-height: var(--DismissLineHeight);
 
-  &:hover {
-    border-color: var(--DismissHoverBorderColor);
-    background-color: var(--DismissHoverBackgroundColor);
-  }
-
+  &:hover,
   &:focus {
-    background-color: var(--DismissFocusBackgroundColor);
-    box-shadow: 0 0 0 2px;
+    background-color: rgba(
+      var(--DismissHoverBackgroundColor),
+      var(--DismissHoverBackgroundOpacity)
+    );
   }
 
   &:disabled {
     background-color: var(--DismissDisabledBackgroundColor);
     color: var(--DismissDisabledTextColor);
   }
-
-  .icon {
```

Comment:
AFAICT this extra wrapper around `Icon` doesn't provide any visual changes. The `Icon` size discrepancy is universal and should be handled within `Icon` since contextual styles won't apply correctly.